### ISSUE TITLE
Tab order navigation inconsistent on My Badges page

### ISSUE THREAD

**Issue by mosefer** [NationalSecurityAgency/skills-service#1104]
If navigating to the My Badges page from the Progress and Ranking view, tab order starts at the Service Desk link as opposed to on My Earned Badges or Available Badges. If the page is navigated to directly (as opposed to from the Progress and Ranking page) the tab order behaves as expected.

**Comment by mosefer** [comment]
cannot reproduce
